src/Ledger/Epoch.agda

```diff
@@ -7,98 +7,108 @@ open import Ledger.Prelude hiding (compare; Rel)
 open import Algebra using (Semiring)
 open import Relation.Binary
 open import Relation.Binary.Definitions using (Total)
-open import Relation.Binary.Consequences using (tri⇒irr)
+open import Relation.Binary.Consequences -- using (tri⇒irr)
 open import Relation.Nullary.Negation
-open import Data.Nat.Properties using (+-*-semiring; ≤-isTotalOrder)
-
+open import Data.Nat.Properties as NatProp using (+-*-semiring; ≤-isTotalOrder; <-isStrictTotalOrder)
 import Data.Nat as ℕ
 
 record EpochStructure : Set₁ where
-  infix 4 _≤ˢ_ _<ˢ_
+  infix 4 _<ˢ_
   field Slotʳ : Semiring 0ℓ 0ℓ
-        Epoch : Set; ⦃ DecEq-Epoch ⦄ : DecEq Epoch
+        Epoch : Set
+        ⦃ DecEq-Epoch ⦄ : DecEq Epoch
+
+  _≡ᵉ?_ : (e e' : Epoch) → Dec(e ≡ e')
+  _ ≡ᵉ? _ = it
 
   Slot = Semiring.Carrier Slotʳ
 
-  field epoch           : Slot → Epoch
```

Comment:
This looks like an accidental reversion to me. Btw, if you want to revert this easily, open this diff in Magit, put your cursor over this hunk (or mark the region you want to revert) and press `v`. If you then amend your commit with the  changes this made, this hunk will disappear.

Reply:
I'm just fixing the alignment here with an extra space.  Would you prefer I leave it unaligned and wait for a purely cosmetic PR to fix alignments?

Reply:
No, we previously agreed to change `_<ˢ_` to `_≤ˢ_`, which you've implemented [here](https://github.com/input-output-hk/formal-ledger-specifications/commit/f3011bb0a4dce5c34aa0c3aa69806d03ab16b2e4#diff-558ffaef3eaece88cf7e59062d1151203aa3a71163d70caa91ea520377866eebL21-R26) and now you're reverting it again.

But also, on the alignment note, this isn't a literate Agda file so there's no point in changing the alignment.

Reply:
Oh, I thought we only temporarily agreed on `_≤ˢ_`, because that made more sense to me and so I could get it working initially, but I thought you still preferred to base everything off `_<ˢ_`, so I switched back, once I could get the latter working.  (That's why this PR's branch is called `...orders-preferred`.)

Okay, I'll change it back... again. 😆

Reply:
No, I prefer `_≤ˢ_` since it's consistent with everything else. However, I believe @omelkonian's suggestion is even better, since it eliminates a bunch of logic from here, and there's indeed no point in giving an explicit name to the inequality here.